cmd/config/errors.go

```diff
@@ -0,0 +1,39 @@
+package config
+
+import "fmt"
+
+const errorReadingYamlFileText = "error trying to read the YAML file: %w"
```

Comment:
I have a big preference for this style: https://github.com/armory-io/deploy-engine/blob/master/cyclops/deployer/errors.go

I'm OK with the helper functions.

Reply:
Also would follow the `ErrMyErrorName` convention: https://github.com/uber-go/guide/blob/master/style.md#error-naming